powerauth-restful-security-spring/src/main/java/io/getlime/security/powerauth/rest/api/spring/service/v3/ActivationService.java

```diff
@@ -341,7 +341,7 @@ public ActivationRemoveResponse removeActivation(PowerAuthApiAuthentication apiA
             response.setActivationId(paResponse.getActivationId());
             return response;
         } catch (Exception ex) {
-            logger.warn("PowerAuth activation removal failed", ex);
+            logger.warn("PowerAuth activation removal failed, error: {}", ex.getMessage());
```

Comment:
Let's pass `ex` as a parameter to the new exception to keep the cause somewhere.